dev/background-api.py

```diff
@@ -0,0 +1,122 @@
+def simulate_toy_bg(alt, az, time):
+    """
+    Function to create dummy bg simulations for test purposes at a particular alt-az pair.
+    Optional: give number of events, instead of observation time.
+    TODO: is there already an established data format?
+
+    Parameters
+    ----------
+    alt : `~astropy.coordinates.Angle`
+        Value of the altitude angle for the simulated observations.
+    az: `~astropy.coordinates.Angle`
+        Value of the azimuth angle for the simulated observations.
+    time : `~astropy.coordinates`
+        Value of the simulated observation time.
+
+    Returns
+    -------
+    image : `~astropy.io.fits.ImageHDU`
+        Image filled with simulated bg events.
+
+    """
+
+
+def simulate_toy_bg(time):
```

Comment:
In Python you can't overload functions like in C++, i.e. you can't have two functions that both have the name `simulate_toy_bg` that just differ by their input arguments. You'd have to use more explicit, different names for the different functions. But before you change things here, let's first have the big-picture discussion how this API spec should look.